helpers/overflow.js

```diff
@@ -0,0 +1,33 @@
+import { unsafeCSS } from 'lit';
+
+export const overflowHiddenDeclarations = getOverflowDeclarations({});
+export const overflowEllipsisDeclarations = getOverflowDeclarations({ textOverflow: 'ellipsis' });
```

Comment:
For your "basic tests", would it make sense to craft some vdiffs for these 3 functions? Not using any of our components, but just some `<div>`s or whatever with pre-canned scenarios where you know it exercises the various bits of logic?